tests/integration/star_wars/star_wars_test_urls.py

```diff
@@ -1,11 +1,11 @@
 STAR_WARS_BASE = '/questionnaire/0/star_wars/789/'
-STAR_WARS_INTRODUCTION = STAR_WARS_BASE + 'introduction'
+STAR_WARS_INTRODUCTION = STAR_WARS_BASE + '14ba4707-321d-441d-8d21-b8367366e766/0/introduction'
```

Comment:
Bit the bullet and sorted them all out under https://github.com/ONSdigital/eq-survey-runner/pull/987/files